week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,9 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = students.every(student => {
+  return group.includes(student) !== true;
```

Comment:
This is correct, but we can make the code a bit shorter. Remember that `.includes()` always returns a boolean. So we don't need to compare it against `true`